src/controllers/dynakube/oneagent/daemonset/daemonset.go

```diff
@@ -26,11 +24,11 @@ const (
 	hostRootVolumeName  = "host-root"
 	hostRootVolumeMount = "/mnt/root"
 
-	certVolumeName  = "certs"
-	certVolumeMount = "/mnt/dynatrace/certs"
+	trustedCaVolumeName  = "dynatrace-cluster-ca"
```

Comment:
```suggestion
	trustedCertVolumeName  = "dynatrace-cluster-cert"
```
I'd suggest using `cert` instead of `ca` in every case since it has more meaning

Reply:
"cert" is more generic term, CA is a specific cert :thinking:.
I'll change it to CaCert.